// process 객체에 uncaughtException 이벤트를 연결
process.on('uncaughException', function(err){
    
});